### ISSUE TITLE
Can't find GLFW3 in preview 4

### ISSUE THREAD

**Issue by ceranco** [Ultz/Silk.NET#122]
# Summary
Even after updating my [project](https://github.com/ceranco/LearnOpenGL.NET) to use `Silk.NET` *version 1.0.0-preview4*, when running on a computer that doesn't have `glfw3` installed, I get the following:

    Unhandled exception. System.NotSupportedException: Couldn't find a suitable windowing platform. You probably forgot to copy a glfw3 library into your bin/Debug or bin/Release folder (this won't be a problem in Preview 4)

# Steps to reproduce
- Platform: Desktop, Windows 10
- Framework Version: .NET Core 3.1.101
1. Create a project with the following `Program.cs`:
```cs
namespace HelloWindow
{
    static class Program
    {
        static void Main(string[] args)
        {
            var options = WindowOptions.Default;
            options.Title = "Hello Window!";

            var window = Window.Create(options);

            window.Run();
        }
    }
}
```

2. Run the project with either `dotnet run -c Release` or `dotnet run -c Debug`.
    * Make sure you don't have a copy of `glfw3.dll` in your path or `Release`/`Debug` directories!

### Comments
I'm really enjoying using `Silk.NET`, thanks for this awesome library 😃

**Comment by Perksey** [comment]
I see no reason why this wouldn't work, so I'll have to test this.

**Comment by Perksey** [comment]
Hmm yeah I've got this too after removing glfw from my PATH. Will investigate.

**Comment by Perksey** [comment]
Okay I've figured out what's going wrong here and it will require some changes in the NativePackages and SuperInvoke repos. Turns out, we weren't able to iterate over the dependencies as Microsoft.Extensions.DependencyModel decided to give up. And even if we did make it that far, we wouldn't have been able to find the damn DLLs anyway as Ultz.Native.GLFW didn't reference the native DLLs (i.e. Ultz.Native.GLFW.Windows.x64)

All in all, it was pretty much a disaster. Will fix soon.
